### ISSUE TITLE
Add storyTITLE to embed config options

### ISSUE THREAD

**Issue by kenara** [coralproject/talk#3492]
It looks like Metascraper doesn't handle JavaScript, so we would have to set up a proxy server to render our pages just to get the story titles (and other metadata, though I'm not sure the other data is used in Coral). The title seems like the most important piece.

The story IDs and story URLs can be added in the embed script - why not the titles? 

I understand that, as _originally posted by @wyattjoh in https://github.com/coralproject/talk/issues/1459#issuecomment-375736089_ ,
#A scraper lowers the burden for adopting the system, as most newsrooms don't have full control over their CMS, they can't push in asset metadata.

But for those of us who can, it would be easier to just set these options in the embed script.

**Comment by kenara** [comment]
Perhaps you are considering this feature request, perhaps not. Either way, I need to get the story titles into my system and I have not succeeded up to now. 

Under admin/configure/advanced > Story creation, I have story scraping 'on' and a URL set to a server that could return the metadata. But it is not getting called by my Coral install on a different machine. 

If the Scraper proxy URL is http://123.456.78.9:8080/coral_endpoint should I expect my Coral server to access it? How should a scraper proxy be set up? Thanks.

**Comment by kenara** [comment]
To get the metadata from my JavaScript single page app into Coral, I just needed to server-render an equivalent page for each story. I set up a simple endpoint that produces a HTML page with only a HEAD element containing the necessary meta tags. All my stories use the same URL pattern so it was a simple matter of swapping out the host of the live website for that of the render server. 

Like so:

![image](https://user-images.githubusercontent.com/4996484/120232232-08bef480-c24b-11eb-8770-c870b18074a6.png)

Just an idea, for what it's worth..
